megamek/src/megamek/client/ui/swing/MechViewPanel.java

```diff
@@ -130,9 +131,30 @@ public void setMech(Entity entity, MechView mechView){
         }  
     }
     
+    public void setMech(Entity entity, TROView troView){
+        reset();
+        txtMek.setText(troView.processTemplate());
+        txtMek.setCaretPosition(0);
+        Image image = FluffImageHelper.getFluffImage(entity);
+        icon = null;
+        if(null != image) {
+            // We don't want this window to be too big, so scale large images
+            if (image.getWidth(this) > DEFAULT_WIDTH)
+            {
+                double aspect_ratio = (float)image.getWidth(this) / image.getHeight(this);
+                image = image.getScaledInstance(DEFAULT_WIDTH, 
+                        (int)(DEFAULT_WIDTH/aspect_ratio), Image.SCALE_FAST);
+            }            
+            icon = new ImageIcon(image);
+            lblMek.setIcon(icon);
+        }  
+    }
+    
     public void setMech(Entity entity) {
         MechView mechView = new MechView(entity, false);
         setMech(entity,mechView);
+//    	TROView troView = TROView.createView(entity, true);
```

Comment:
This intended as a TODO?

Reply:
Obsolete test code that I've now removed.